go/pkg/cosmos/txhistory.go

```diff
@@ -19,6 +22,39 @@ import (
 	"github.com/shapeshift/go-unchained/pkg/tendermint/client"
 )
 
+// cursor stores state client side between paginated requests
+type Cursor struct {
+	LastBlockHeight     uint64 `json:"blockHeight"`
```

Comment:
Why not just use `int` for number representations generally speaking?

Reply:
thought here is block height is a function of block time, and using unsigned ints where appropriate gives you access to a lot more numbers before you overflow. maybe we should go to BigInt with a custom marshal/unmarshal?